### DISCUSSION TITLE
Save post as a draft and send later

### DISCUSSION THREAD

**Discussion by ouboub** [magit/forge#728] [Ideas]
Hi

I am used,  in gnus, to send an unfinished message to the draft group so that I can continue later and send it away.
I can't find anything similar for issues. So it seems that I can only send them or discard them but not save them if I want to continue later.
Is this correct? If so, could this feature be implemented?
thanks
Uwe Brauer

**Comment by tarsius** [comment]
You can abort with `C-c C-k`, that saves the draft. The next time you begin writing a "new" message of the same type in the same repository, you will be given the option to resume that draft or start over.

**Comment by ouboub** [comment]
ok thanks!

**Reply by tarsius** [reply]
In the future I might add support for saving multiple drafts and for *showing* whether there are drafts, instead of the current "try and see" approach. So I am converting this back to an "idea", instead of a "q/a" discussion.

**Reply by ouboub** [reply]
> In the future I might add support for saving multiple drafts and for _showing_ whether there are drafts, instead of the current "try and see" approach. So I am converting this back to an "idea", instead of a "q/a" discussion.

right, that would be fantastic, because actually sometimes, I have to stop in the middle of writing an issue, so having such draft features would be clearly an enhancement!

**Reply by pkryger** [reply]
> In the future I might add support for saving multiple drafts and for *showing* whether there are drafts, instead of the current "try and see" approach. So I am converting this back to an "idea", instead of a "q/a" discussion.

That would be great. I use the existing functionality pretty often but more than once I was burned by switching to high-priority-emergency-fix-asap, and clobbering a draft for the original work.
